model/src/main/java/org/cbioportal/model/ClinicalAttribute.java

```diff
@@ -0,0 +1,61 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+
+public class ClinicalAttribute implements Serializable {
```

Comment:
Would you consider using lombok's [@Data](https://projectlombok.org/features/Data.html)? lombok adds some overhead to the build process but overall it does a great job a cutting down the java boilerplate. (PS: there's also a immutable counterpart to `@Data` called `@Value`)